contrib/libmfx/module.defs

```diff
@@ -6,7 +6,9 @@ LIBMFX.FETCH.sha256 = e07d1024e86998ac3992620f5db0f999af51cc700f7de90da2ebbe1e8a
 
 LIBMFX.CONFIGURE.bootstrap = rm -fr aclocal.m4 autom4te.cache; autoreconf -fiv;
 
+ifneq (1,$(FEATURE.qsv))
```

Comment:
I'm confused about what this is supposed to do.  The libmfx contrib is only built when FEATURE.qsv == 1.  So it would appear this will never add --without-libva_\* since you can only get here if FEATURE.qsv == 1.  Maybe this conditional should be checking BUILD.system instead? E.g.
> ifneq (,$(filter $(BUILD.system),linux))